backend/Schema/auth/signup.js

```diff
@@ -0,0 +1,35 @@
+const schema = {
+  required: ['username', 'email_id', 'name', 'password', 'department'],
+  properties: {
+    username: {type: 'string'},
+    email_id: {type: 'string', format: 'email'},
+    name: {type: 'string'},
+    password: {type: 'string', format: 'password'},
+    branch: {type: 'number'},
+    department: {type: 'number'},
+    admission_no: {type: 'string', minLength: 8, maxLength: 8},
+    semester: {type: 'number', minLength: 1, maxLength: 8},
+  },
+  errorMessage: {
```

Comment:
This will not work if you're using only ajv. ajv-errors need to be installed too.